pulp_2_tests/tests/puppet/api_v2/test_sync_publish.py

```diff
@@ -178,19 +191,22 @@ def test_non_matching_query(self):
 
     def _sync_repo(self, repo):
         """Sync a repository, and verify no tasks contain an error message."""
-        report = sync_repo(self.cfg, repo).json()
-        for task in api.poll_spawned_tasks(self.cfg, report):
+        cfg = config.get_config()
+        report = sync_repo(cfg, repo).json()
+        for task in api.poll_spawned_tasks(cfg, report):
             self.assertIsNone(
                 task['progress_report']['puppet_importer']['metadata']['error_message']  # pylint:disable=line-too-long
             )
 
 
-class SyncInvalidFeedTestCase(BaseAPITestCase):
-    """If an invalid feed is given a sync should complete with errors."""
+class SyncInvalidFeedTestCase(unittest.TestCase):
+    """If an invalid feed is given, a sync should complete with errors."""
 
     @classmethod
     def setUpClass(cls):
         """Create a puppet repository with an invalid feed and sync it."""
+        cls.cfg = config.get_config()
+        cls.resources = set()
         super().setUpClass()
```

Comment:
From the previous comments, this one can be removed.

Reply:
Yea, that was a miss. Thanks.